### PR TITLE
Update update-dns-records-to-retain-current-hosting-provider.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the steps for Updating customer records

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2210

File: microsoft-365/admin/dns/update-dns-records-to-retain-current-hosting-provider.md [hunk 1]
```diff
@@ -32,9 +32,11 @@ description: "Learn how to route traffic to an existing public website hosted ou
 ## Update DNS records in the Microsoft 365 admin center
 1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834818" target="_blank">Domains</a> page.
 
-2. On the **Domains** page, in the list of domains, select the domain you're using for your website, and then select **DNS settings** in the management pane. 
-    
-3. Select **+ New custom record** and enter the following: 
+2. On the **Domains** page, select the domain and then choose **DNS Records**.
+
+3. Under **DNS settings**, select **Custom Records**,
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/dns/update-dns-records-to-retain-current-hosting-provider.md]:
```suggestion
3. Under **DNS settings**, select **Custom Records**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**